NOTICE.txt

```diff
@@ -786,6 +846,10 @@ THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
 
 ---
 
+## intl
```

Comment:
I can't find it here https://www.npmjs.com/package/@types/highlight.js

Also seems that it was removed with PR https://github.com/mattermost/mattermost-webapp/commit/ca91a49d2e55c9f39ace13109a2d8ace7c015959#diff-b9cfc7f2cdf78a7f4b91a753d10865a2, but then re-added via https://github.com/mattermost/mattermost-webapp/commit/55035969cf0cade5784851a4623b34f04e1b01d8#diff-b9cfc7f2cdf78a7f4b91a753d10865a2.

Reply:
Hmmm. Looks like a casualty of @hmhealey's merge issue.

Reply:
You can remove it. It will be removed again soon.